### ISSUE TITLE
2.0.0-rc3 release

### ISSUE THREAD

**Issue by jsha** [algesten/ureq#271]
Let's do another release today!

**Comment by algesten** [comment]
Yea!

**Comment by ffleader1** [comment]
So I am confused about how to read the response body in version 2.0.0-rc3. Was it ```.into_reader()``` or ```.into_string()```?

I have the following function:

```
fn func4_get_req(){
    let mut request = ureq::get("https://maps.googleapis.com/maps/api/staticmap");

    let response = request
        .query("center", "Booklyn+Bridge,New+York,NY")
        .query("zoom", "13")
        .query("size", "600x300")
        .query("key", "AIza****")
        .call();
    println!("{:?}",response);

}
```
Printing the response would just result in printing the status:
```Ok(Response[status: 200, status_text: OK])```
Which means the request was successful but... nothing else. 
Adding either ```response.into_string()``` or ```response.into_reader()``` would just result in another error:
```method not found in `std::result::Result<ureq::Response, ureq::Error>```
So I am not sure how to handle it. 
This is how the command execute in Postman:
![image](https://user-images.githubusercontent.com/15016720/101991526-e290a280-3c72-11eb-9189-0b6add5080a7.png)
It gets a picture, so I am not sure if that makes a difference.

**Comment by jsha** [comment]
@ffleader1 try this:

```rust
fn func4_get_req(){
    let mut request = ureq::get("https://maps.googleapis.com/maps/api/staticmap");

    let response: String = request
        .query("center", "Booklyn+Bridge,New+York,NY")
        .query("zoom", "13")
        .query("size", "600x300")
        .query("key", "AIza****")
        .call()?
        .into_string()?;
    println!("{:?}",response);
}
```

**Comment by jsha** [comment]
Although, having posted that, I realize you don't want `into_string()`, since this is binary data - you want `into_reader()`, and then read that reader into a Vec.

**Comment by jyn514** [comment]
I think this can be closed now that 2.0 is published.
